whirlpool/appliance.py

```diff
@@ -122,6 +128,21 @@ def bool_to_attr_value(self, b: bool):
     def attr_value_to_bool(self, val: str):
         return None if val is None else val == SETVAL_VALUE_ON
 
+    def attr_value_to_int(self, val: str):
+        value = 0
+        try:
+            value = int(val)
+        except ValueError:
+            return 0
```

Comment:
I think we should return None if it is not an in, otherwise we will have no way to distinguish an invalid value from a valid zero one upper in the lib or HA.